src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -6,26 +6,7 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import seedu.address.logic.commands.AddCommand;
-import seedu.address.logic.commands.ClearCommand;
-import seedu.address.logic.commands.Command;
-import seedu.address.logic.commands.CopyCommand;
-import seedu.address.logic.commands.DeleteCommand;
-import seedu.address.logic.commands.EditCommand;
-import seedu.address.logic.commands.ExitCommand;
-import seedu.address.logic.commands.FindCommand;
-import seedu.address.logic.commands.FindCommandLetter;
-import seedu.address.logic.commands.FindEmailCommand;
-import seedu.address.logic.commands.FindPhoneCommand;
-import seedu.address.logic.commands.FindTagCommand;
-import seedu.address.logic.commands.HelpCommand;
-import seedu.address.logic.commands.HistoryCommand;
-import seedu.address.logic.commands.ListCommand;
-import seedu.address.logic.commands.ListTagsCommand;
-import seedu.address.logic.commands.RedoCommand;
-import seedu.address.logic.commands.SelectCommand;
-import seedu.address.logic.commands.SortCommand;
-import seedu.address.logic.commands.UndoCommand;
+import seedu.address.logic.commands.*;
```

Comment:
IIRC, we were told to avoid using wildcards and only to import the ones we used. Perhaps unless we imported everything under commands, we shouldn't use the wildcard?